versioninfo.json.tmpl

```diff
@@ -0,0 +1,30 @@
+{{- $name := "chezmoi" -}}
+
+{{- $commitHash := output "git" "rev-parse" "HEAD" | trim -}}
+{{- if ne "" (output "git" "diff" "--stat" | trim) -}}
+{{-   $commitHash = printf "%s-dirty" $commitHash -}}
+{{- end -}}
+
+{{- $version := semver (output "git" "describe" "--abbrev=0" "--tags" | trim) -}}
+{{- $versionDict := dict "Major" $version.Major "Minor" $version.Minor "Patch" $version.Patch "Build" 0 -}}
+{{- $fFileVersion := dict "FileVersion" $versionDict -}}
+{{- $fProductVersion := dict "ProductVersion" $versionDict -}}
+{{- $fixedFileInfo := dict "FixedFileInfo" (merge $fFileVersion $fProductVersion) -}}
+
+{{- $filename := printf "%s.exe" $name -}}
+
+{{- /* Get the current year using the Magical Reference Date format */ -}}
+{{- $currentYear := now | date "2006" -}}
+
+{{- $sComments := dict "Comments" (printf "commit: %s" $commitHash) -}}
+{{- $sFileDescription := dict "FileDescription" "Manage your dotfiles across multiple diverse machines, securely." -}}
+{{- $sFileVersion := dict "FileVersion" $version.Original -}}
+{{- $sInternalName := dict "InternalName" $filename -}}
+{{- $sLegalCopyright := dict "LegalCopyright" (printf "Copyright (c) %s Tom Payne" $currentYear) -}}
```

Comment:
The copyright year needs to be 2018 to `$currentYear`, i.e. please replace `%s` with `2018-%s`.

Reply:
Done.